modules/javafx.controls/src/test/java/test/com/sun/javafx/scene/control/infrastructure/MouseEventFirer.java

```diff
@@ -60,6 +60,9 @@ public final class MouseEventFirer {
     public MouseEventFirer(EventTarget target) {
         this.target = target;
 
+        // Use the alternative creation path for MouseEvent by default, see JDK-8253769
```

Comment:
If the alternative path is made the default then this class needs some cleanup. That can be done as part of `JDK-8253769`. Request you to add a comment in JBS on `JDK-8253769` to clean up the unused portion of `MouseEventFirer.java`